tucan/tests.py

```diff
@@ -0,0 +1,31 @@
+import networkx as nx
```

Comment:
Can we rename this to `test_utils`? `tests` (imo wrongly) implies that the tests live _here_, in this module, not outside.